docs/content/how-to/baremetal-agent.md

```diff
@@ -0,0 +1,643 @@
+# HyperShift + Bare Metal workers using 'agent' provider
+
```

Comment:
can this be consolidated with https://github.com/openshift/hypershift/blob/main/docs/content/how-to/agent-platform.md?

Reply:
PTAL